terrahelp/mask.go

```diff
@@ -2,6 +2,7 @@ package terrahelp
 
 import (
 	"fmt"
+	"github.com/acarl005/stripansi"
```

Comment:
package used to strip out the ANSI escape codes from strings.  Terraform 0.12.x produces coloured output.